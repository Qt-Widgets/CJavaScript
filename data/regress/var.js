var i;
var sum;
console.log(i);
console.log(sum);

var i, sum;
console.log(i);
console.log(sum);

var i = 2;
console.log(i);
console.log(sum);
