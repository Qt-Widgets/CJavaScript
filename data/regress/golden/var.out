undefined
undefined
undefined
undefined
2
undefined
